src/scrapers/otsar-hahayal.ts

```diff
@@ -94,42 +94,19 @@ function convertTransactions(txns: ScrapedTransaction[]): Transaction[] {
 }
 
 async function parseTransactionPage(page: Page): Promise<ScrapedTransaction[]> {
-  const tdsValues = await pageEvalAll<{ classList: string, innerText: string}[]>(page, '#dataTable077 tbody tr td', [], (tds) => {
-    return (tds as HTMLElement[]).map((td) => ({
-      classList: td.getAttribute('class') || '',
-      innerText: (td).innerText || '',
+  const tdsValues = await pageEvalAll(page, '#dataTable077 tbody tr', [], (trs) => {
+    return (trs).map((el) => ({
+      date: (el.querySelector('.date') as HTMLElement).innerText,
```

Comment:
Not sure if there is a better way to define this type `HTMLElement` in one place instead of in each field.
If there is, Let me know and I will fix it...

Reply:
I think it is ok like that for this scenario